cloudman/values.yaml

```diff
@@ -178,9 +178,17 @@ cloudlaunch:
       oidc_public_uri: "{{.Values.ingress.protocol }}://{{ .Values.global.domain | default (index .Values.ingress.hosts 0) }}/cloudman"
     postgresql:
       enabled: true
-      postgresDatabase: cloudman
-      postgresUser: cloudman
-      postgresPassword: some_pass
+      postgresqlDatabase: cloudman
+      postgresqlUsername: cloudman
+      postgresqlPassword: some_pass
+      #postgresqlPostgresPassword: admin_pass
```

Comment:
This line can go?

Reply:
I just left it to be visible if we ever need a superuser password for postgres. By default the `postgres` user doesn't have a password. I just used another init container in cloudman so it wasn't needed for the keycloak DB, but it might be helpful to have the option later?